web/users.go

```diff
@@ -58,7 +62,21 @@ func (u *userAuthRoundtrip) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	user, pass, auth := r.BasicAuth()
 	if auth {
 		if hashedPassword, ok := c.Users[user]; ok {
-			if err := bcrypt.CompareHashAndPassword([]byte(hashedPassword), []byte(pass)); err == nil {
+			cacheKey := hex.EncodeToString(append(append([]byte(user), []byte(hashedPassword)...), []byte(pass)...))
```

Comment:
We could also sha512 this one maybe if we don't want plaintext in memory.

Reply:
Basic auth is plain text anyway, so I'm not that concerned about it.